js-exercises/telephone-check/telephoneCheck.js

```diff
@@ -0,0 +1,12 @@
+
+const telephoneCheck = () => {
+  if(typeof phoneNumber !== 'string') {
+    throw TypeError(`Expected string. got ${typeof phoneNumber}`);
+  }
+  const re = new RegExp(/^(1\s?)?((\([0-9]{3}\))|[0-9]{3})[\s\-]?[\0-9]{3}[\s\-]?[0-9]{4}$/)
+  return re.test(phoneNumber)
```

Comment:
The code looks good. But the variable name can be improved like instead of "re" you could use something like regex, pattern, etc..

Reply:
ok. i will make the changes